lib/tasuku.rb

```diff
@@ -13,6 +13,7 @@ def configure
 
     def config
       @config ||= Configatron::Store.new author: :current_user
+      @config ||= Configatron::Store.new update_answers: false
       @config
     end
```

Comment:
Yeah, `Configatron::Store.new` just accepts a hash of keys and values that describe defaults.